src/app/app-routing.module.ts

```diff
@@ -25,21 +25,22 @@ const appRoutes: Routes = [
         component: DeviceComponent
       },
       {
-        path: 'devices/:id', 
+        path: 'devices/:id',
         component: DeviceListComponent
       },
       {
         path: 'newlocation/:id',
         component: NewLocationComponent
       }
-    ] 
+    ]
   },
   { path: '**', component: PageNotFoundComponent }
 ];
 
 @NgModule({
   imports: [
-    RouterModule.forRoot(appRoutes, { useHash: true })
```

Comment:
Keeping this causes the anchor tags to fail—instead of jumping to the specified spot on the same page it tries to redirect to a different page.